crates/astria-conductor/src/block_verifier.rs

```diff
@@ -70,13 +69,17 @@ impl BlockVerifier {
             hex::encode(data.block_hash),
         );
 
-        let current_validator_set = client
-            .validators(height, sequencer_client::tendermint_rpc::Paging::Default)
+        // the validator set which votes on a block at height `n` is the
+        // set at `n-1`. the validator set at height `n` is the set after
+        // the results of block `n`'s execution have occurred.
+        let validator_set = client
```

Comment:
Could add an info log that we are fetching the validator set for the prev height.